scripts/get_nodejs_bin

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+NODEJS_7x_LATEST_URL="https://nodejs.org/dist/latest-v7.x"
+NODEJS_WIN_x64="win-x64"
+NODEJS_WIN_x86="win-x86"
+NODEJS_LINUX_x64="linux-x64"
+NODEJS_LINUX_x86="linux-x86"
+# TODO : This is temporal fixed version. Need to get latest or specific version from dist repository
+NODEJS_LATEST7X_VERSION="node-v7.4.0"
+
+source ./scripts/path_info
+
+# Get the nodeJS binary for running platform
+# $1 : platform name
+function get_nodejs_binary() {
+  platform_name=$1
+  if [ "$platform_name" = "linux_x86_64" ]; then
+    target_node_platform_name="$NODEJS_LATEST7X_VERSION-$NODEJS_LINUX_x64"
+    target_node_archive_file="$target_node_platform_name.tar.gz"
+  elif [ "$platform_name" = "linux_x86" ]; then
+    target_node_platform_name="$NODEJS_LATEST7X_VERSION-$NODEJS_LINUX_x86"
+    target_node_archive_file"$target_node_platform_name.tar.gz"
+  elif [ "$platform_name" = "windows_x86_64" ]; then
+    target_node_platform_name="$NODEJS_LATEST7X_VERSION-$NODEJS_WIN_x64"
+    target_node_archive_file"$target_node_platform_name.zip"
+  elif [ "$platform_name" = "windows_x86" ]; then
+    target_node_platform_name="$NODEJS_LATEST7X_VERSION-$NODEJS_WIN_x86"
+    target_node_archive_file"$target_node_platform_name.zip"
+  fi
+
+  target_url="$NODEJS_7x_LATEST_URL/$target_node_archive_file"
+
+  target_node_dir=$(get_node_dir)
+
+  donwload_file_to_path "$target_url" "$target_node_dir"
+  if [ ! "$?" = "0" ]; then
+    return
+  fi
+
+  extract_archive "$target_node_dir/$target_node_archive_file" \
+  "$target_node_dir"; rm -f $target_node_dir/$target_node_archive_file
+
+  # FIXME(Jaecule) : This workaround is for removing dir
+  # of archive file need more graceful way
+  mv $target_node_dir/$target_node_platform_name/* "$target_node_dir/"
+  rmdir "$target_node_dir/$target_node_platform_name"
+}
+
+# Get the download file to given path
+# $1 : download url
+# $2 : file to locate
+# Out: 0 if success to donwload file else negative
+function donwload_file_to_path() {
+  url=$1
+  path=$2
+  if [ -z "$1" ] || [ -z "$2" ]; then
+    echo "Invalid donwload url or path url : $1 path : $2"
+    return -1
+  fi
+
+  echo "download from $target_url to $target_path"
+
+  if [ ! -d "$target_path" ]; then
+    echo "targetPath does not exist. create $target_path"
+    mkdir $target_path
+  fi
+
+  wget $url -P $path
+  if [ "$?" != "0" ]; then
+    echo "Failed to download"
+    return -2
+  fi
+
+  return 0
+}
+
+# extract archive files to given path
+#
+# Arguments:
+# $1 : path of archive
+# $2 : path of uncompressed files
+# Remarks : if $2 is not given, uncompress files to current path
+function extract_archive() {
+    if [ -z "$2" ]; then
```

Comment:
These code(83line ~ 87line) can be simplified as follows:
  - dest="${2-./}"

But it's not mandatory. If you think your code is better for readability, you don't need to fix this.